contents/handbook/growth/sales/yc-onboarding.md

```diff
@@ -82,6 +82,18 @@ You should:
 - After they've gotten set up, [create a deal on the appropriate Hubspot pipeline](/handbook/growth/sales/sales-operations)
 - Add their email domain to the 'YC W21 Onboarded Users' cohort as a new match group that tags `email contains @theirdomain.com`
 
+## Processing AirPods and Allbirds orders
+
+We collect responses to the [the merch form](https://forms.gle/K61bhD6uLxaaTqoK6) and process these manually once a week by placing the orders on the user's behalf. You can just place the order on your Brex card. 
+
+We provide the standard [AirPods](https://www.apple.com/shop/product/MV7N2AM/A/airpods-with-charging-case) and case (ie. not AirPods Pro or the wireless charging case). For Allbirds, we provide any shoes that show as $95 or less on their [US site](https://www.allbirds.com/). 
+
+For users in a country which has an Apple or Airpods website, just use the relevant country's site. This usually applies to the USA, UK and EU. 
+
+For anywhere else, you should ask the user to purchase the product themselves and then send us their receipt and bank account details for reimbursement. It is not a good use of time to try and navigate resellers or e-commerce sites in a language you don't understand, especially as they probably won't accept a US card anyway. 
```

Comment:
I know we did this but I think the standard here was supposed to be (and should be) a gift card for the equivalent amount

Reply:
it's _often_ cheaper to buy from the UK or US amazon using the company card, so I'd start there first usually>